app/jobs/import_versions_job.rb

```diff
@@ -1,5 +1,26 @@
+require 'httparty'
+
+
+class ElasticLogger
+  include HTTParty
+  base_uri ENV.fetch('ELASTICSEARCH_URL', 'http://localhost:9200')
+
+  def report_size(size)
+    response = self.class.post(
+      '/import_queue/_doc/',
```

Comment:
Hmmm, I think we need something to support the different queues, though. Would we just post to a different URL for each queue?